lib/pages/submission_page.dart

```diff
@@ -0,0 +1,234 @@
+          children: [
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: Text("Submit a found item", style: theme.darkTitleStyle),
+            ),
+            IconButton(
+                color: theme.dark,
+                icon: const Icon(Icons.keyboard_return_rounded),
+                onPressed: () => routeBack(context))
+          ],
+        );
+
+    Widget buildForm() => FastForm(
+            inputDecorationTheme: InputDecorationTheme(
+              disabledBorder: const OutlineInputBorder(
+                borderSide: BorderSide(width: 3),
+              ),
+              enabledBorder: OutlineInputBorder(
+                borderSide: BorderSide(color: theme.medium, width: 3),
+              ),
+              focusedBorder: const OutlineInputBorder(
+                borderSide: BorderSide(width: 3),
+              ),
+              errorBorder: OutlineInputBorder(
+                borderSide: BorderSide(color: theme.error, width: 3),
+              ),
+              focusedErrorBorder: OutlineInputBorder(
+                borderSide: BorderSide(color: theme.error, width: 2),
+              ),
+              filled: true,
+              fillColor: theme.light,
+            ),
+            formKey: formKey,
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: FastDropdown(
+                  initialValue: "Other",
+                  name: "item_name_dropdown",
+                  items: itemNameOptions,
+                  onChanged: (value) => setState(() => itemName = value!),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: FastTextField(
+                  maxLines: 5,
+                  name: "description_text_field",
+                  labelText: "Description",
+                  validator: (String? value) {
+                    if (value == null || value.isEmpty) {
+                      return 'Please enter some text';
+                    }
+                    description = value;
+                    return null;
+                  },
+                ),
+              ),
+            ]);
+
+    Widget buildUploadButton() => Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Container(
+            decoration: BoxDecoration(
+                color: theme.light,
+                border: Border.all(color: theme.medium, width: 3),
+                borderRadius: const BorderRadius.all(Radius.circular(3))),
+            child: TextButton.icon(
+                icon: const Icon(Icons.upload),
+                label: Text(bytes == null ? "Upload image" : "Change image"),
+                onPressed: () => loadImageIntoMemory()),
+          ),
+        );
+
+    Widget buildSubmitButton() => loading
+        ? const Loading()
+        : IconButton(
+            icon: const Icon(Icons.add_box_rounded),
```

Comment:
This icon isn't really explanatory. Could we keep the button with the word 'Submit' so it clearly indicated as such?